curve-ansible/roles/generate_config/defaults/main.yml

```diff
@@ -190,6 +190,9 @@ client_mds_rpc_retry_interval_us: 100000
 client_metacache_get_leader_timeout_ms: 500
 client_metacache_get_leader_retry: 5
 client_metacache_rpc_retry_interval_us: 100000
+client_mds_normal_retry_times_before_trigger_wait: 3
+client_mds_max_wait_ms: 86400000
```

Comment:
> the number of mds max wait ms is different between clent.conf and this.

The config in `client.conf` is for robot test, if we use the real value (like `86400000`), it will retry for long time, and the test case will timeout and failed.

Reply:
Yeap, it's useful for unit test, but i think it's unuseful for robot test, like [curve_robot_test](http://59.111.91.248:8080/job/opencurve_multijob/1294/).

Reply:
I have confirmed that the robot test will use the config file under the conf directory. @wu-hanqing @cw123

Reply:
> I have confirmed that the robot test will use the config file under the conf directory. @wu-hanqing @cw123

Which test case causes robot test timeout?

Reply:
> > I have confirmed that the robot test will use the config file under the conf directory. @wu-hanqing @cw123
> 
> Which test case causes robot test timeout?

The case for `read write offset > size`, [click for detail](https://github.com/opencurve/curve/blob/master/robot/curve_robot.txt#L1364).

Reply:
If you, you should distinguish whether IO request is exceeded file length or allocate segment failed caused by no space left in the logical pool.

Reply:
yeap, it's my neglect, i should ignore other errors except **NO SPACE** error.